clients/python/README.md

```diff
@@ -3,7 +3,9 @@
 - Python 3
 
 ## Setup
-`pip install -r requirements.txt`
+1. `pip install -r requirements.txt`
+2. Rename `player.default.json` to `player.json` and replace the
+   `team_secret` by the value that was given to you by the organiser
 
 ## Joining the Ranked Game
 `python play.py`
```

Comment:
`play.py` has to be accessible, so you would do `python play.py` if it is in the same folder (`clients/python`). You could just do `python ./clients/python/play.py` too.